### DISCUSSION TITLE
Nice SphereOS with Windows Style :D

### DISCUSSION THREAD

**Discussion by DeafMan1983** [Jspa2/SphereOS#8] [General]
Hey I am surprised that how did you use GUI? Excellent work! I would like know how did you make realistic operating system and it was written in C#.

Nice and keep your work! I love to develop own operating system :+1: 
Is Sphere OS an initial operating system written from C#?  :-D
Enjoy your development!

**Comment by jspa2** [comment]
Thank you for your support!

It is fully written in C# using the [Cosmos](https://github.com/cosmosos/cosmos) operating system development framework. The GUI is based Cosmos' included graphics drivers.

**Comment by DeafMan1983** [comment]
Yeah I would like to develop with Cosmos and I try out. But I use Ubuntu 22.04 and dotnet 6.0.13. I will look for cosmos development. Welcome and I would like to support your operating system if I have idea.
